docs/usage/configuration/config_documentation.md

```diff
@@ -3716,8 +3775,11 @@ instance_map:
 ### `stream_writers`
 
 Experimental: When using workers you can define which workers should
-handle event persistence and typing notifications. Any worker
-specified here must also be in the `instance_map`.
+handle write streams such as event persistence and typing notifications.
```

Comment:
Perhaps "handling writing to streams" here>

Reply:
=> https://github.com/matrix-org/synapse/pull/14086/commits/a214ee89e8c0702c585897484a58fd5b9c94f0fd

Reply:
Oops, typo from me. I should have said "handle writing to streams". I'll fixup

Reply:
8394c3eb7d